### ISSUE TITLE
Realize native ReadWriteMany connection mode for LINSTOR volumes

### ISSUE THREAD

**Issue by EvgenySamoylov** [deckhouse/deckhouse#5115]
### Preflight Checklist

- [X] I agree to follow the [Code of Conduct](https://github.com/deckhouse/deckhouse/blob/main/CODE_OF_CONDUCT.md) that this project adheres to.
- [X] I have searched the [issue tracker](https://github.com/deckhouse/deckhouse/issues) for an issue that matches the one I want to file, without success.

### Use case. Why is this important?

Some applications while running on different nodes, still need to be able to read and write data to a single volume.
If we use LINSTOR as a cluster storage solution, we don't have a legit way for ReadWriteMany connection mode out of the box since LINSTOR doesn't support it.
In other words, the volume cannot be mounted as read-write by many nodes.

### Proposed Solution

Lets consider to propose some solution to ReadWriteMany volumes connection mode out of the box with LINSTOR.
More investigation is needed here.
LINBIT recommends the [following approach](https://kb.linbit.com/how-can-i-achieve-rwx-for-linstor-volumes-which-are-rwo).

**Comment by konstantin-axenov** [comment]
You have to use appropriate storage for your applications. If you need RWX than you can't use linstor.
